### PR TITLE
en: add docs for generate --fail-on-error

### LINKED ISSUE #1635: Document nuxt generate --fail-on-error
Document https://github.com/nuxt/nuxt.js/pull/5195

### PR DESCRIPTION
**Pull request by runyasak**
Fix #1635

I have update document about `nuxt generate` with `--fail-on-page-error`.
If you have any feedback or suggestion, please let me know and I will resolve. 

Thanks 😁

### CODE REVIEW

File: en/guide/commands.md [hunk 1]
```diff
@@ -100,6 +100,12 @@ npm run generate
 
 It will create a `dist` folder with everything inside ready to be deployed on a static hosting site.
 
+To see error status from page generation, you need to pass `--fail-on-page-error` argument to command:
```
[the remark this answers is not in the document]
Comment (runyasak) [reply en/guide/commands.md]: Oh, sorry for my misunderstand.
I will resolve it. 😂

File: en/guide/commands.md [hunk 2]
```diff
@@ -100,6 +100,16 @@ npm run generate
 
 It will create a `dist` folder with everything inside ready to be deployed on a static hosting site.
 
+To return non-zero status code when a page error is encountered and let CI cancel deployment, you need to pass `--fail-on-page-error` argument to command.
```
Comment (manniL) [inline en/guide/commands.md]:
```suggestion
To return a non-zero status code when a page error is encountered and let the CI/CD fail the deployment or build, you can use the `--fail-on-page-error` argument. 
```

Comment (runyasak) [reply en/guide/commands.md]: That's my terrible grammar.
I will fix it. 😂

**Comment by manniL** [comment]
thanks!

**Pull request merged**